src/publisher/api_v2_views.py

```diff
@@ -18,12 +18,15 @@
 
 ERR = "error"
 
+CURRENT_POA_VERSION = str(settings.ALL_SCHEMA_IDX["poa"][0][0])
+CURRENT_VOR_VERSION = str(settings.ALL_SCHEMA_IDX["vor"][0][0])
+
 
 def ctype(status):
     return {
-        POA: "application/vnd.elife.article-poa+json; version=2",
-        VOR: "application/vnd.elife.article-vor+json; version=3",
-        ERR: "application/json",  # text/plain as well?
+        POA: "application/vnd.elife.article-poa+json; version=" + CURRENT_POA_VERSION,
```

Comment:
I assume string conversion of `CURRENT_POA_VERSION` is absolutely safe here if it's an `int`?

Reply:
strings can be concatenated with `+` in python.

`CURRENT_POA_VERSION` is defined just above this section of code and is wrapped in a call to `str`.

Should that `str` ever be removed any test calling a view (that in turn calls `ctype`) would fail immediately.